backend/controllers/user.controller.ts

```diff
@@ -236,84 +168,38 @@ export const loginUser = async (
       },
     });
 
-    if (existingUser) {
-      if (existingUser.isVerified === false) {
-        const otp = otpGenerator.generate(6, {
-          upperCaseAlphabets: false,
-          lowerCaseAlphabets: false,
-          specialChars: false,
-        });
-
-        await prisma.user.update({
-          where: {
-            email: email,
-          },
-          data: {
-            otp: otp,
-          },
-        });
-
-        // send user an email with new otp
-        const message = {
-          from: "nomansanjari2001@gmail.com",
-          to: email,
-          subject: "Verify OTP - CampusBuddy",
-          html: `<b>${otp}</b>`,
-        };
-
-        await transporter.sendMail(message);
-
-        res.status(100).json({
-          success: false,
-          message: "User not verified, otp sent",
-        });
-      } else if (existingUser.isVerified) {
-        const token = jwt.sign(
-          { ID: existingUser.id },
-          process.env.JWT_SECRET ?? "testSecret",
-        );
-
-        await prisma.user.update({
-          where: {
-            email: email,
-            password: password,
-          },
-          data: {
-            jwt: token,
-          },
-        });
-
-        // send jwt to client
-        res.status(200).cookie("authToken", token).json({
-          success: true,
-          message: "JWT set",
-        });
-      }
-    } else {
+    if (!existingUser) {
       res.status(404).json({
         success: false,
-        message: "User not found",
+        message: "User doesn't exist",
+      });
+    } else {
+      const token = jwt.sign(
+        {
+          ID: existingUser.id,
+          institutionID: existingUser.institutionId,
+          username: existingUser.username,
+          firstName: existingUser.firstName,
+          lastName: existingUser.lastName,
+          email: existingUser.email,
+          password: existingUser.password,
+        },
+        process.env.JWT_SECRET ?? "testSecret",
+      );
+
+      res.status(200).cookie("token", token).json({
+        success: true,
+        message: "Login successful",
       });
     }
   } catch (error) {
     next(error);
   }
 };
 
-// logout Student -> protected route
+// logout Student
 export const logoutUser = async (req: Request, res: Response) => {
-  const { email } = emailSchema.parse(req.body);
```

Comment:
do we even need this route anymore? The front end just needs to purge the JWT token, no request is needed right? 

@jcharlpretorius @JamesRobert20

Reply:
yes it could do that. I'll leave it unchanged for now seeing as no one has said anything yet

Reply:
Okay sounds good, in the future we might even want to blacklist the JWT